lib/ui5Framework/npm/Registry.js

```diff
@@ -49,13 +49,22 @@ class Registry {
 	}
 	async _getPacoteOptions() {
 		if (!this._npmConfig) {
-			const {default: libnpmconfig} = await import("libnpmconfig");
-			const opts = {
-				cache: this._cacheDir,
-			};
-			const config = libnpmconfig.read(opts, {
-				cwd: this._cwd
-			}).toJSON();
+			const {default: Config} = await import("@npmcli/config");
+			const {
+				default: {flatten, definitions, shorthands},
+				defaults,
+			} = await import("@npmcli/config/lib/definitions/index.js");
+
+			const configuration = new Config({
+				npmPath: this._cwd,
+				definitions,
+				flatten,
+				shorthands,
+				defaults
+			});
+
+			await configuration.load(); // Reads through the configurations
+			const config = configuration.flat; // JSON. Formatted via "flatten"
 
 			// Rename https-proxy to httpsProxy so that it is picked up by npm-registry-fetch (via pacote)
 			if (config["https-proxy"]) {
```

Comment:
What I can see is that in `npm-registry-fetch` it's still `httpsProxy` and `pacote` doesn't do anything about it. `@npmcli/config` still defines it as `https-config`: https://github.com/npm/cli/blob/latest/workspaces/config/lib/definitions/definitions.js#L875

Reply:
Yes, because that's how the configuration is supposed to be defined in the .npmrc.
But this differs from my observations, where "https-proxy" from the .npmrc is now correctly read as httpsProxy, so I don't see a need for this special handling anymore.

Reply:
I see :) 
Removed it